module-2/backend/week_one.md

```diff
@@ -5,30 +5,132 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+*Get - displays a page at a given route*
+
+*Post - places new information into a database*
+
+*Put - updates existing information in a database*
+
+*Patch - updates select attributes*
+
+*Delete - removes existing information from a database*
+
 2. What is Sinatra?
+
+*Sinatra is a domain specific language used to create web apps using Ruby.*
+
 4. What is MVC?
+
+*Models, Views, and Controllers. Models keep a relationship with the database and store/validate data. Views present data to users in a specific format, in our case Embedded Ruby (ERB). Controllers direct traffic, pull data from models and organize it to fit the specifictions of a given view.* 
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+*The conventions create a uniform directory structure that's easy for other developers to use.*
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+*Instance variables*
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
+    name  = "Mr. Ed"
+    
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
 9. What's the purpose of ERB?
+
+*ERB embeds Ruby code in HTML files*
+
 10. Why do I need a development AND test database?
+
+*You want to try to isolate your development database so as to not affect your tests if something goes wrong in development.*
+
 11. What's responsive design?
+
+*Responsive design refers to rendering a web page based on the size of a screen/window.*
+
 12. What is CRUD and why is it important?
-13. What does HTTP stand for? 
+
+*Create, Read, Update, Delete. These are the basic interactions users have with a web app.*
+
+13. What does HTTP stand for?
+
+*Hypertext transfer protocol*
+
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+
+*<% %> executes the code between the within the brackets and <%= %> displays ERB views*
+
 15. What's an ORM?
+
+*I know it stands for Object Relational Mapping and in ActiveRecord it has something to do with models interacting with a database but that's all I know.*

```

Comment:
ORM is the more general term. ActiveRecord is an example of an ORM. It maps relations in our database to OO models we can use in our code.